grand_geckos/ui.py

```diff
@@ -7,6 +7,8 @@
 
 
 class EditorFrame(Frame):
+    box_data: str
```

Comment:
You are using a class wide variable to store the box data.

How about using a dataclass instance that you pass into both Frames as an argument?